samples/iree-reduce/interesting.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Maybe better to put this in a `test` directory.

Reply:
Also, would be more portable if it was a python script (even if canonical usage is to use a shell script for brevity, python would be more testable)